spec/features/fee_type_spec.rb

```diff
@@ -2,34 +2,32 @@
 
 RSpec.feature "fee_type" do
   let(:user) { users(:user) }
+  let(:fee_type) { fee_types(:upgrade) }
 
   before :each do
     visit "/"
     sign_in user
-
-    @fee_type = FeeType.new(name: 'Upgrade Membership', amount: '2000')
-    @fee_type.save!
   end
 
   scenario 'SHOW' do
     visit fee_types_path
-    click_link @fee_type.name
+    click_link fee_type.name
 
     expect(page).to have_content('Upgrade Membership')
     expect(page).to have_content('20.00')
```

Comment:
From the spec here there is no way to see why the page would have '20.00' or 'Upgrade Membership' in it's content. Usually with factory bot i'd, i'd explicitly set those values in the spec before inserting them (I know that is not possible here).

Would it be better in this case actually to write `expect(page).to have_content(fee_type.name)`...

Reply:
100% agree...I was concentrated just to fix/replace fixtures didn't have to to look inside `expectations`...will push the correction tomorrow morning.